shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -98,10 +101,18 @@
               + "default is 8MB when copying from local, "
               + "and 64MB when copying to local")
           .build();
+  private static final Option PRESERVE_OPTION =
+      Option.builder("p")
+          .longOpt("preserve")
+          .required(false)
+          .desc("Preserve file permission attributes when copying files. "
+              + "All ownership, permissions and ACLs will be preserved")
+          .build();
 
   private int mCopyFromLocalBufferSize = COPY_FROM_LOCAL_BUFFER_SIZE_DEFAULT;
   private int mCopyToLocalBufferSize = COPY_TO_LOCAL_BUFFER_SIZE_DEFAULT;
   private int mThread = Runtime.getRuntime().availableProcessors() * 2;
+  private boolean mPreserveAttributes = false;
```

Comment:
`mPreservePermissions`? Want to avoid using "attributes" to avoid confusing regarding ttl, etc

Reply:
renamed.